airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/util/S3OutputPathHelper.java

```diff
@@ -30,8 +30,7 @@ public static String getOutputPrefix(final String bucketPath, final String names
       paths.add(NAME_TRANSFORMER.convertStreamName(namespace));
     }
     paths.add(NAME_TRANSFORMER.convertStreamName(streamName));
-
-    return String.join("/", paths);
+    return String.join("/", paths).replaceAll("/+", "/");
```

Comment:
please add unit test in S3OutputPathHelperTest class with empty namespace to prevent accidently removing of this option

Reply:
Yes, of course, already added two cases for S3OutputPathHelperTest